config/lint/lint.gradle

```diff
@@ -0,0 +1,14 @@
+android {
+    lintOptions {
+        lintConfig file("$project.rootDir/config/lint/lint-config.xml")
+
+        warningsAsErrors true
+        abortOnError true
+
+        xmlReport false
+        htmlOutput file("$project.buildDir/reports/lint/lint.html")
+
+        // disable lint because it does not work well with java8 lambda functions
```

Comment:
Unfortunately, I don't know :S This is the Lint file from our project. We can maybe delete it. Try it out and then let me now :)